<h1>2020 USA Presidential Voter Intentions Analysis</h2>

During the 2020 U.S. presidential election, voter turnout became a major point of discussion. Many Americans chose not to vote, citing logistical barriers, a lack of trust in the electoral system, and dissatisfaction with the candidates. However, some voters were more motivated than ever, feeling that the stakes were personal or that they needed to avoid potential regret. Various demographic groups, such as younger voters and those from lower-income backgrounds, faced more significant obstacles to voting. https://projects.fivethirtyeight.com/non-voters-poll-2020-election/

This analysis will explore the factors influencing voter participation, highlighting key trends and insights from the data to better understand voter behavior in the 2020 election. Data source available on https://github.com/fivethirtyeight/data/tree/master/non-voters

In [2]:
import pandas as pd
import os
import numpy as np
df=pd.read_csv('nonvoters_data.csv')
df.head(5)

RespId  weight  Q1  Q2_1  Q2_2  Q2_3  Q2_4  Q2_5  Q2_6  Q2_7  ...  Q30  \
0  470001  0.7516   1     1     1     2     4     1     4     2  ...    2   
1  470002  1.0267   1     1     2     2     3     1     1     2  ...    3   
2  470003  1.0844   1     1     1     2     2     1     1     2  ...    2   
3  470007  0.6817   1     1     1     1     3     1     1     1  ...    2   
4  480008  0.9910   1     1     1    -1     1     1     1     1  ...    1   

   Q31  Q32  Q33  ppage                 educ   race  gender     income_cat  \
0  NaN  1.0  NaN     73              College  White  Female       $75-125k   
1  NaN  NaN  1.0     90              College  White  Female  $125k or more   
2  NaN  2.0  NaN     53              College  White    Male  $125k or more   
3  NaN  1.0  NaN     58         Some college  Black  Female        $40-75k   
4 -1.0  NaN  NaN     81  High school or less  White    Male        $40-75k   

   voter_category  
0          always  
1          always  
2        sporadic  
3        sporadic  
4          always  

[5 rows x 119 columns]

In [197]:
df.shape

(5836, 119)

In [198]:
df.columns.values

array(['RespId', 'weight', 'Q1', 'Q2_1', 'Q2_2', 'Q2_3', 'Q2_4', 'Q2_5',
       'Q2_6', 'Q2_7', 'Q2_8', 'Q2_9', 'Q2_10', 'Q3_1', 'Q3_2', 'Q3_3',
       'Q3_4', 'Q3_5', 'Q3_6', 'Q4_1', 'Q4_2', 'Q4_3', 'Q4_4', 'Q4_5',
       'Q4_6', 'Q5', 'Q6', 'Q7', 'Q8_1', 'Q8_2', 'Q8_3', 'Q8_4', 'Q8_5',
       'Q8_6', 'Q8_7', 'Q8_8', 'Q8_9', 'Q9_1', 'Q9_2', 'Q9_3', 'Q9_4',
       'Q10_1', 'Q10_2', 'Q10_3', 'Q10_4', 'Q11_1', 'Q11_2', 'Q11_3',
       'Q11_4', 'Q11_5', 'Q11_6', 'Q14', 'Q15', 'Q16', 'Q17_1', 'Q17_2',
       'Q17_3', 'Q17_4', 'Q18_1', 'Q18_2', 'Q18_3', 'Q18_4', 'Q18_5',
       'Q18_6', 'Q18_7', 'Q18_8', 'Q18_9', 'Q18_10', 'Q19_1', 'Q19_2',
       'Q19_3', 'Q19_4', 'Q19_5', 'Q19_6', 'Q19_7', 'Q19_8', 'Q19_9',
       'Q19_10', 'Q20', 'Q21', 'Q22', 'Q23', 'Q24', 'Q25', 'Q26', 'Q27_1',
       'Q27_2', 'Q27_3', 'Q27_4', 'Q27_5', 'Q27_6', 'Q28_1', 'Q28_2',
       'Q28_3', 'Q28_4', 'Q28_5', 'Q28_6', 'Q28_7', 'Q28_8', 'Q29_1',
       'Q29_2', 'Q29_3', 'Q29_4', 'Q29_5', 'Q29_6', 'Q29_7', 'Q29_8',
 

All the answers are numbered based, following the pdf layout on GitHub's link. <br>
If a question header is followed by underline and another number, means a subquestion.<br>
&emsp;For example, the 2nd question (Q2) is about how important each subquestion is to being considered a good American.<br>
&emsp;Than, the first subquestion, Q2_1, is voting in elections, which can receive a answer from 1 to 4, which 1 is very important, and 4 is not at all important.<br>
&emsp;This answer scale can changes for each question.<br>
Some questions only are asked for some given former answer.

<hr>

<h3>1. Sample characteristics</h3>

In [112]:
# age information
df['ppage'].describe().astype(int)

count    5836
mean       51
std        17
min        22
25%        36
50%        54
75%        65
max        94
Name: ppage, dtype: int32

In [40]:
# education level, race, gender, income category and voter category informations:
educ=df['educ'].value_counts(normalize=True).round(2).to_dict()
df_educ = pd.DataFrame([educ]).T.reset_index()
race=df['race'].value_counts(normalize=True).round(2).to_dict()
df_race = pd.DataFrame([race]).T.reset_index()
gender=df['gender'].value_counts(normalize=True).round(2).to_dict()
df_gender = pd.DataFrame([gender]).T.reset_index()
income_cat=df['income_cat'].value_counts(normalize=True).round(2).to_dict()
df_income_cat = pd.DataFrame([income_cat]).T.reset_index()
voter_category=df['voter_category'].value_counts(normalize=True).round(2).to_dict()
df_voter_category = pd.DataFrame([voter_category]).T.reset_index()

df_characteristics= pd.DataFrame()
df_characteristics = pd.concat([df_educ,df_characteristics,df_race,df_gender,df_income_cat,df_voter_category],axis=1,ignore_index=True)
df_characteristics.rename(columns={ 0:'Educational_level',1:'%',2:'Race',3:'%',4:'Gender',5:'%',
                                    6:'Income_group',7:'%',8:'Voter_category',9:'%' },
                          inplace=True)
df_characteristics

Educational_level     %         Race     %  Gender    %    Income_group  \
0              College  0.40        White  0.64    Male  0.5        $75-125k   
1  High school or less  0.31        Black  0.16  Female  0.5  Less than $40k   
2         Some college  0.29     Hispanic  0.14     NaN  NaN         $40-75k   
3                  NaN   NaN  Other/Mixed  0.07     NaN  NaN   $125k or more   

      % Voter_category     %  
0  0.28       sporadic  0.44  
1  0.24         always  0.31  
2  0.24   rarely/never  0.25  
3  0.24            NaN   NaN

The sample has 5.839 individuals (lines) with age average of 51 years old and well distributed.<br>Gender and education are nearly equal distributed, 63% are white, 44% vote sporadically, 31% vote always and 25% vote rarely/never

<h3>1.1 Race x income categories</h3>

- What's the income category most representative in each race of the sample?

In [39]:
df_race_income_valuecounts=pd.DataFrame()
df_race_income_valuecounts = df[['race','income_cat']].value_counts().reset_index()
df_race_income_valuecounts = df_race_income_valuecounts.merge(df_race_income_valuecounts.groupby('race')['count'].sum(),on='race')
df_race_income_valuecounts.rename(columns={ 'count_x':'Count', 
                                            'count_y':'Total_count_race',
                                            'race':'Race','income_cat':
                                            'Income_category','count':'Count'},inplace=True)
df_race_income_valuecounts['%_within_race'] = (df_race_income_valuecounts['Count'] / df_race_income_valuecounts['Total_count_race']).round(2)
df_race_income_valuecounts = df_race_income_valuecounts.groupby('Race').first()
df_race_income_valuecounts.sort_values(by='%_within_race',ascending=False,inplace=True)
df_race_income_valuecounts

Income_category  Count  Total_count_race  %_within_race
Race                                                               
Black        Less than $40k    334               932           0.36
Other/Mixed   $125k or more    126               381           0.33
White              $75-125k   1070              3710           0.29
Hispanic           $75-125k    231               813           0.28

 - Within the races, what's the proportion of people belonging to the wealthiest group, with income higher than $125k?

In [9]:
df_race_income_valuecounts[df_race_income_valuecounts['income_cat'] == '$125k or more'].sort_values('%_within_race',ascending=False)

race     income_cat  count  total_race  %_within_race
12  Other/Mixed  $125k or more    126         381           0.33
1         White  $125k or more    983        3710           0.26
10     Hispanic  $125k or more    158         813           0.19
11        Black  $125k or more    127         932           0.14

- Within the races, what's the proportion of people belonging to the poorest group, with income less than $40k?

In [10]:
df_race_income_valuecounts[df_race_income_valuecounts['income_cat'] == 'Less than $40k'].sort_values('%_within_race',ascending=False)

race      income_cat  count  total_race  %_within_race
4         Black  Less than $40k    334         932           0.36
8      Hispanic  Less than $40k    221         813           0.27
3         White  Less than $40k    784        3710           0.21
14  Other/Mixed  Less than $40k     79         381           0.21

<h3>1.2 Education x income categories</h3>

- What's the most representative income group in each education group?

In [36]:
df_educ_income_valuecounts = df[['educ','income_cat']].value_counts().reset_index()
df_educ_income_valuecounts = df_educ_income_valuecounts.merge(df_educ_income_valuecounts.groupby('educ')['count'].sum(),on='educ')
df_educ_income_valuecounts.rename(columns={'count_x': 'Count', 
                                           'count_y':'Total_educ',
                                           'educ':'Educational_level',
                                           'income_cat': 'Income_category'},inplace=True)
df_educ_income_valuecounts['%_within_educ'] = (df_educ_income_valuecounts['Count']/df_educ_income_valuecounts['Total_educ']).round(2)
df_educ_income_valuecounts = df_educ_income_valuecounts.groupby('Educational_level').first()
df_educ_income_valuecounts.sort_values(by='%_within_educ',ascending=False,inplace=True)
df_educ_income_valuecounts

Income_category  Count  Total_educ  %_within_educ
Educational_level                                                    
High school or less  Less than $40k    782        1796           0.44
College               $125k or more    954        2330           0.41
Some college               $75-125k    511        1710           0.30

<hr>

<h3>2. Based on the questions asked:</h3>

- The voters were splited into always/sporadic/rarely-never voters.<br>How are their distribution among educational level?

In [205]:
df_educ_voter_valuecounts = df[['educ','voter_category']].value_counts().reset_index()
df_educ_voter_valuecounts = df_educ_voter_valuecounts.merge(df_educ_voter_valuecounts.groupby('educ')['count'].sum(),on='educ')
df_educ_voter_valuecounts.rename(columns={'count_x': 'Count', 'count_y':'Total_educ',
                                          'educ':'Educational_level', 'voter_category': 'Voter_category'},inplace=True)
df_educ_voter_valuecounts['%_within_educ'] = (df_educ_voter_valuecounts['Count']/df_educ_voter_valuecounts['Total_educ']).round(2)
df_educ_voter_valuecounts.sort_values(by='%_within_educ',ascending=False,inplace=True)
df_educ_voter_valuecounts.groupby('Voter_category').first().sort_values(by='%_within_educ',ascending=False)

Educational_level  Count  Total_educ  %_within_educ
Voter_category                                                       
sporadic                    College   1073        2330           0.46
always                      College    834        2330           0.36
rarely/never    High school or less    631        1796           0.35

- How are party preferences by educational level?

In [203]:
df_educ_party_valuecounts = df[['Q30','educ']].value_counts().reset_index()
df_educ_party_valuecounts['Q30'] =  np.where(df_educ_party_valuecounts['Q30']==1,'Republican',
                                        np.where(df_educ_party_valuecounts['Q30']==2,'Democrat',
                                        np.where(df_educ_party_valuecounts['Q30']==3,'Independent',
                                        np.where(df_educ_party_valuecounts['Q30']==4,'Another party',
                                        np.where(df_educ_party_valuecounts['Q30']==5,'No preference',
                                        np.where(df_educ_party_valuecounts['Q30']==-1,'NA',''))))))

df_educ_party_valuecounts = df_educ_party_valuecounts.merge(df_educ_party_valuecounts.groupby('Q30')['count'].sum(),on='Q30')
df_educ_party_valuecounts.rename(columns={'count_x': 'Count', 
                                          'count_y':'Total_party', 
                                          'Q30': 'Party_preference',
                                          'educ': 'Educational_level'},
                                 inplace=True)
df_educ_party_valuecounts['%_within_party'] = (df_educ_party_valuecounts['Count']/df_educ_party_valuecounts['Total_party']).round(2)
df_educ_party_valuecounts.sort_values(by=['Party_preference','%_within_party'],ascending=[True,False],inplace=True)
df_educ_party_valuecounts.groupby('Party_preference').first().sort_values(by='%_within_party',ascending=False)        

Educational_level  Count  Total_party  %_within_party
Party_preference                                                         
NA                High school or less     24           48            0.50
Another party                 College     49           99            0.49
No preference     High school or less    326          670            0.49
Independent                   College    679         1425            0.48
Democrat                      College    900         2002            0.45
Republican                    College    563         1592            0.35

- How many people that supports Democrat and Republican will vote in the opposite party candidate?

In [207]:
df_voting_preference = df[['Q30','Q23']].value_counts().reset_index()
df_voting_preference['Q23'] =  np.where(df_voting_preference['Q23']==1,'Trump',
                                        np.where(df_voting_preference['Q23']==2,'Biden',
                                        np.where(df_voting_preference['Q23']==3,'Incert',
                                        np.where(df_voting_preference['Q23']==-1,'NA',''))))

df_voting_preference['Q30'] =  np.where(df_voting_preference['Q30']==1,'Republican',
                                        np.where(df_voting_preference['Q30']==2,'Democrat',
                                        np.where(df_voting_preference['Q30']==3,'Independent',
                                        np.where(df_voting_preference['Q30']==4,'Another party',
                                        np.where(df_voting_preference['Q30']==5,'No preference',
                                        np.where(df_voting_preference['Q30']==-1,'NA',''))))))

df_voting_preference = df_voting_preference.merge(df_voting_preference.groupby('Q23')['count'].sum(),on='Q23')
df_voting_preference.rename(columns={'count_x': 'Count', 'count_y':'Total_party',
                                    'Q23': 'Intention_to_vote', 'Q30': 'Party_preference'},inplace=True)
df_voting_preference['%_within_party'] = (df_voting_preference['Count']/df_voting_preference['Total_party']).round(2)
df_voting_preference[((df_voting_preference['Party_preference'] == 'Democrat') & (df_voting_preference['Intention_to_vote'] == 'Trump'))
                    | ((df_voting_preference['Party_preference'] == 'Republican') & (df_voting_preference['Intention_to_vote'] == 'Biden'))]

Party_preference Intention_to_vote  Count  Total_party  %_within_party
10       Republican             Biden     95         2777            0.03
11         Democrat             Trump     45         1907            0.02

- Does it matter who will win the election based on the country issues? How is the opinion by party preference?

In [209]:
df_perspective = df[['Q30','Q5']].value_counts().round(2).reset_index()
df_perspective['Q5'] = np.where (df_perspective['Q5'] == 1, 'Who wins the election matters',
                        np.where(df_perspective['Q5'] == 2, 'Things will be pretty much the same',
                        np.where(df_perspective['Q5'] == -1, 'NA','')))
df_perspective['Q30'] =  np.where(df_perspective['Q30']==1,'Republican',
                            np.where(df_perspective['Q30']==2,'Democrat',
                            np.where(df_perspective['Q30']==3,'Independent',
                            np.where(df_perspective['Q30']==4,'Another party',
                            np.where(df_perspective['Q30']==5,'No preference',
                            np.where(df_perspective['Q30']==-1,'NA',''))))))

df_perspective = df_perspective.merge(df_perspective.groupby('Q30')['count'].sum(),on='Q30')
df_perspective.rename(columns={'count_x': 'Count', 'count_y':'Total_count_party',
                              'Q30': 'Party_preference',
                              'Q5': 'Perspective of election result'},
                      inplace=True)
df_perspective['%_within_party'] = (df_perspective['Count']/df_perspective['Total_count_party']).round(2)
df_perspective.sort_values(by=['Party_preference','%_within_party'],ascending=[True,False],inplace=True)
df_perspective.groupby('Party_preference').first().sort_values('%_within_party',ascending=False)

Perspective of election result  Count  Total_count_party  \
Party_preference                                                            
Democrat          Who wins the election matters   1820               2002   
Republican        Who wins the election matters   1402               1592   
Independent       Who wins the election matters   1129               1425   
Another party     Who wins the election matters     64                 99   
No preference     Who wins the election matters    370                670   
NA                Who wins the election matters     25                 48   

                  %_within_party  
Party_preference                  
Democrat                    0.91  
Republican                  0.88  
Independent                 0.79  
Another party               0.65  
No preference               0.55  
NA                          0.52

- How important is displaying the American flag to being a good American?

In [160]:
df['Q2_4'].value_counts(normalize=True).round(2).rename(index={1:'Very important',2:'Somewhat important',3:'Not so important',4:'Not at all important',-1:'NA'})

Q2_4
Very important          0.34
Somewhat important      0.28
Not so important        0.23
Not at all important    0.15
NA                      0.01
Name: proportion, dtype: float64

- How important is displaying the American flag to being a good American by education level (showing only the most representative importance opinion by each education level)?

In [41]:
df_educ_flag_valuecounts = df[['educ','Q2_4']].value_counts().rename(index={1:'Very important',2:'Somewhat important',3:'Not so important',4:'Not at all important',-1:'NA'}).reset_index()
df_educ_flag_valuecounts = df_educ_flag_valuecounts.merge(df_educ_flag_valuecounts.groupby('educ')['count'].sum(),on='educ')
df_educ_flag_valuecounts.rename(columns={'count_x': 'Count', 'count_y':'Total_educ',
                                         'educ': 'Education_level',
                                         'Q2_4': 'Importance_opinion'},inplace=True)
df_educ_flag_valuecounts['%_within_educ'] = (df_educ_flag_valuecounts['Count']/df_educ_flag_valuecounts['Total_educ']).round(2)
df_educ_flag_valuecounts = df_educ_flag_valuecounts.groupby('Education_level').first()
df_educ_flag_valuecounts.sort_values(by='%_within_educ',ascending=False,inplace=True)
df_educ_flag_valuecounts

Importance_opinion  Count  Total_educ  %_within_educ
Education_level                                                         
High school or less     Very important    813        1796           0.45
Some college            Very important    634        1710           0.37
College               Not so important    693        2330           0.30

- How is the most representative importance opinion about displaying the American flag to being a good American by race?

In [221]:
df_race_flag_valuecounts = df[['race','Q2_4']].value_counts().rename(index={1:'Very important',2:'Somewhat important',3:'Not so important',4:'Not at all important',-1:'NA'}).reset_index()
df_race_flag_valuecounts = df_race_flag_valuecounts.merge(df_race_flag_valuecounts.groupby('race')['count'].sum(),on='race')
df_race_flag_valuecounts.rename(columns={'count_x': 'Count', 'count_y':'Total_race', 'race': 'Race',
                                        'Q2_4': 'Importance_opinion'},inplace=True)
df_race_flag_valuecounts['%_within_race'] = (df_race_flag_valuecounts['Count']/df_race_flag_valuecounts['Total_race']).round(2)
df_race_flag_valuecounts.sort_values(by=['Race','%_within_race'],ascending=[True,False],inplace=True)
df_race_flag_valuecounts.groupby('Race').first().sort_values(by='%_within_race',ascending=False)

Importance_opinion  Count  Total_race  %_within_race
Race                                                            
White           Very important   1427        3710           0.38
Hispanic        Very important    281         813           0.35
Black         Not so important    291         932           0.31
Other/Mixed   Not so important    114         381           0.30

- How important is displaying the American flag to being a good American by party preference (showing only the most representative importance opinion by each party preference)?

In [217]:
df_party_flag_valuecounts = df[['educ','Q30']].value_counts().reset_index()
df_party_flag_valuecounts['Q30'] =  np.where(df_party_flag_valuecounts['Q30']==1,'Republican',
                                        np.where(df_party_flag_valuecounts['Q30']==2,'Democrat',
                                        np.where(df_party_flag_valuecounts['Q30']==3,'Independent',
                                        np.where(df_party_flag_valuecounts['Q30']==4,'Another party',
                                        np.where(df_party_flag_valuecounts['Q30']==5,'No preference',
                                        np.where(df_party_flag_valuecounts['Q30']==-1,'NA',''))))))
                                                
df_party_flag_valuecounts = df_party_flag_valuecounts.merge(df_party_flag_valuecounts.groupby('Q30')['count'].sum(),on='Q30')
df_party_flag_valuecounts.rename(columns={'count_x': 'Count', 'count_y':'Total_party',
                                         'Q30': 'Party_preference',
                                         'educ': 'Educational_level'},inplace=True)
df_party_flag_valuecounts['%_within_party'] = (df_party_flag_valuecounts['Count']/df_party_flag_valuecounts['Total_party']).round(2)
df_party_flag_valuecounts.sort_values(by=['Party_preference','%_within_party'],ascending=[True,False],inplace=True)
df_party_flag_valuecounts.groupby('Party_preference').first().sort_values(by='%_within_party',ascending=False)                                        

Educational_level  Count  Total_party  %_within_party
Party_preference                                                         
NA                High school or less     24           48            0.50
Another party                 College     49           99            0.49
No preference     High school or less    326          670            0.49
Independent                   College    679         1425            0.48
Democrat                      College    900         2002            0.45
Republican                    College    563         1592            0.35

- How important is believing in God important to being a good American?

In [218]:
df['Q2_9'].value_counts(normalize=True).round(2).rename(index={
                                                    1:'Very important',
                                                    2:'Somewhat important',
                                                    3:'Not so important',
                                                    4:'Not at all important',
                                                    -1:'NA'})

Q2_9
Very important          0.47
Not at all important    0.22
Somewhat important      0.16
Not so important        0.14
NA                      0.01
Name: proportion, dtype: float64

- How important is believing in God important to being a good American by educational level?

In [45]:
df_educ_god_valuecounts = df[['educ','Q2_4']].value_counts().rename(index={1:'Very important',2:'Somewhat important',3:'Not so important',4:'Not at all important',-1:'NA'}).reset_index()
df_educ_god_valuecounts = df_educ_god_valuecounts.merge(df_educ_god_valuecounts.groupby('educ')['count'].sum(),on='educ')
df_educ_god_valuecounts.rename(columns={'count_x': 'Count', 'count_y':'Total_educ',
                                        'Q2_4':'Importance_opinion',
                                        'educ': 'Educational_level'},inplace=True)
df_educ_god_valuecounts['%_within_educ'] = (df_educ_god_valuecounts['Count']/df_educ_god_valuecounts['Total_educ']).round(2)
df_educ_god_valuecounts = df_educ_god_valuecounts.groupby('Educational_level').first()
df_educ_god_valuecounts.sort_values(by='%_within_educ',ascending=False,inplace=True)
df_educ_god_valuecounts

Importance_opinion  Count  Total_educ  %_within_educ
Educational_level                                                       
High school or less     Very important    813        1796           0.45
Some college            Very important    634        1710           0.37
College               Not so important    693        2330           0.30

- People who didn't have intention to vote in 2020 (5,8k interviewed)

In [91]:
df_intention_countvalues = df['Q21'].value_counts(normalize=True).round(2).rename(index={1: 'Yes', 2: 'No', 3: 'Unsure/Undecided', -1: 'NA'})
df_intention_countvalues.rename('Do you intend to vote in 2020 elections?')

Q21
Yes                 0.85
No                  0.08
Unsure/Undecided    0.07
NA                  0.00
Name: Do you intend to vote in 2020 elections?, dtype: float64

- Only for who answered No in the last question: why didn't they intend to vote?

In [259]:
df_intention_countvalues = df['Q22'].value_counts(normalize=True).round(2).rename(index={
    1: "Doesn't have time to register or vote", 
    2: "Doesn't trust the political system to serve their needs",
    3: "Doesn't know how to register",
    4: "Doesn't want to register",
    5: 'Not eligible to vote',
    6: "Doesn't think their vote matters",
    7: 'Other',
    -1: 'NA'})
df_intention_countvalues

Q22
Doesn't want to register                                   0.24
Doesn't trust the political system to serve their needs    0.22
Doesn't think their vote matters                           0.21
Other                                                      0.18
Doesn't have time to register or vote                      0.06
Not eligible to vote                                       0.04
Doesn't know how to register                               0.04
NA                                                         0.02
Name: proportion, dtype: float64

<hr>

<h3>Conclusions</h3>

- The sample has 5,836 interviewed individuals, with an average age of 51. The sample was almost evenly split between genders, with 63% identifying as White, 40% holding a college degree, and nearly evenly divided into four income groups, with the wealthiest earning above $125k and the poorest earning under $40k.<br>
- 44% of the sample vote sporadically, 31% always, and 25% rarely or never.<br>
- The "Other/mixed race" race has the wealthiest income group (above 125k) as the most representative. Black category is the only one where the most representative group earns less than $40k.<br>
- Among the most representative income categories within each educational level, the college graduates are the wealthiest, while the less educated group is the poorest across all educational levels.<br>
- Voters who vote rarely or never are the only group where the lowest educational level is the most representative.<br>
- Among all party preference groups, Republicans have the lowest percentage of college graduates, at 35%, followed by Democrats at 45%.<br>
- More Republicans intended to vote for Biden than Democrats did for Trump.<br>
- 91% of Democratic voters believe that the election result is important, compared to 88% of Republicans.<br>
- For 34% of those interviewed, displaying the American flag is a very important sign of being a good American. White individuals, the least educated group, and Republicans are the most likely to agree with this.<br>
- For 47% of those interviewed, believing in God is a very important sign of being a good American. The least educated groups are more likely to agree with this.<br>
- Among people who did not intend to vote in 2020 (8%), 22% did not plan to vote because they did not believe that the political system would serve their needs, and 21% did not think their vote mattered.<br>